Classes/Middleware/Captcha.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Blueways\BwCaptcha\Middleware;
+
+use Blueways\BwCaptcha\Utility\CaptchaBuilderUtility;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use TYPO3\CMS\Core\Routing\PageArguments;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManager;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
+use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
+use TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController;
+
+class Captcha implements MiddlewareInterface
+{
+    protected ResponseFactoryInterface $responseFactory;
+    protected ConfigurationManager $configurationManager;
+    protected TypoScriptFrontendController $tsfe;
+
+    public function __construct(
+        ResponseFactoryInterface $responseFactory,
+        ConfigurationManager $configurationManager,
+        TypoScriptFrontendController $tsfe,
```

Comment:
PHP parse error, because of comma at the end

Reply:
fixed.